README.md

```diff
@@ -125,6 +148,12 @@ People and organizations on GitHub who have multiple repositories related to che
 ## Miscellaneous
 * [ASEE Chemical Engineering Division](http://ched.asee.org/)
 * [The History of Chemical Engineering](https://www.youtube.com/watch?v=aRKyJRAxjpM&ab_channel=CrashCourse) - CrashCourse YouTube video
+* [Drug discovey with AI](https://github.com/benstaf/ChemGAN-challenge)
```

Comment:
I'm not sure that any of these belong on this repo, except maybe the PID tuning. The rest are all interesting (and are closely related to my personal research area), but I don't think they fit under the chemical engineering education umbrella. These would be better suited to other lists like [awesome-python-chemistry](https://github.com/lmmentel/awesome-python-chemistry/) or [awesome-cheminformatics](https://github.com/hsiaoyi0504/awesome-cheminformatics) linked below. The field of ML for molecules is massive, and if we start including too much in that area, this list could become too long to be useful.